cmd/ndm_daemonset/probe/mountprobe_test.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package probe
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"syscall"
+	"testing"
+
+	"github.com/openebs/node-disk-manager/pkg/mount"
```

Comment:
please rearrange the imports

Reply:
Is there a specific fomat we follow? I've been relying on the default auto formatting of my code editor.

Reply:
The priority is as follows:
1. Golang standard packages like, io, strings etc
2. Packages from vendor
3. Packages from same repo

This is the format that we need, and slowly we are moving towards this.